src/server/config.ts

```diff
@@ -3,6 +3,7 @@ export interface IServerConfig {
   port: string | number;
   responseDelay: number;
   protos: IProto[];
+  randomResponses: boolean;
```

Comment:
Currently, this value is never set.
Could you please rename this
```suggestion
  matchedResponses: boolean;
```
and then init it in the commands:
https://github.com/dushanpantic/grpc-server-mock/blob/master/src/commands/StartAutoWireCommand.ts
https://github.com/dushanpantic/grpc-server-mock/blob/master/src/commands/GenerateConfigCommand.ts
(I can do this part later tonight also)